dashboard/app/models/script.rb

```diff
@@ -2024,9 +2024,13 @@ def get_unit_resources_pdf_url
     end
   end
 
+  def middle_high?
+    csd? || csp? || csa?
+  end
```

Comment:
I think Dani is working on something that will make this check more direct, maybe check with her to see if this is right way to check for middle/high now or if there's something else we can use?

Reply:
@dmcavoy Is there a preferred way to group these? I didn't see any other helper methods that did quite what I wanted, but am also open to other suggestions!

Reply:
We don't have any better way to check these yet but the course types conversation should hopefully lead to something better here. Would you mind adding a TODO for me to come back here and update this once we have figured out course types?

Reply:
Added! Thanks, Dani.